### PR TITLE
add floatThead

### PR DESCRIPTION
**Pull request by mkoryak**
this thing https://github.com/mkoryak/floatThead since cdnjs forgot how to auto update itself a few months ago

### CODE REVIEW

**Comment by megawac** [comment]
Hi, could you included an `update.json` file so our auto updater is enabled for your project (details in Contributing.md)

**Comment by mkoryak** [comment]
I promise to add it after you merge this, i really don't want to pull down this repo

**Comment by megawac** [comment]
Feel free to use the webgui to add the file :) (go on github to your branch
and add file)

**Comment by mkoryak** [comment]
poop, should find a better example

File: files/floatthead/update.json-packageManager-github-name-floatthead-repo-mkoryak/floatThead-files-basePath-dist/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (megawac) [inline files/floatthead/update.json-packageManager-github-name-floatthead-repo-mkoryak/floatThead-files-basePath-dist/update.json]: File belongs under
`files/floatthead/update.json`

**Comment by mkoryak** [comment]
i did something bad there. i am just gonna clone it and do it the way i always do it

**Pull request closed without merging**